ci-operator/jobs/openstack-k8s-operators/designate-operator/openstack-k8s-operators-designate-operator-main-presubmits.yaml

```diff
@@ -1,5 +1,67 @@
 presubmits:
   openstack-k8s-operators/designate-operator:
+  - agent: kubernetes
+    always_run: true
```

Comment:
Set this to `false`
Together with skip_if_only_changed mentioned below, this job will always run and skip when PR only have irrelevant fles